packages/xrpl/test/models/baseTransaction.test.ts

```diff
@@ -78,7 +78,7 @@ describe('BaseTransaction', function () {
     assert.throws(
       () => validateBaseTransaction(invalidFee),
       ValidationError,
-      'BaseTransaction: invalid Fee',
+      'Payment: invalid field Fee',
```

Comment:
Why was this changed from `BaseTransaction` to `Payment`?

Reply:
The error was changed when I refactored the `validateBaseTransaction` method to use the new helper functions.